couchdb/test.couchdb-cluster.yml

```diff
@@ -80,6 +63,6 @@ services:
       - "TEST_MODE=CLUSTERED"
       - "COUCHDB_USER=${COUCHDB_USER:-admin}"
       - "COUCHDB_PASSWORD=${COUCHDB_PASSWORD:-password}"
-      - "SVC_NAME=haproxy"
+      - "SVC_NAME=couchdb.1"
```

Comment:
Does this mean that the test will only hit one of the nodes?

Reply:
No, I've removed HAProxy from the test setup. We are only testing if the cluster sets up correctly. The tests are hardcoded to each node and retrieve data from each. We will just send the first call to see if `couchdb is ready` to a particular node instead of haproxy, which would need a slight rewrite of the tests to work.

Reply:
Cool, thanks for explaining!!